bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubHandler.java

```diff
@@ -180,23 +176,36 @@ public synchronized NeoHubReturnResult toNeoHubSendChannelValue(String commandSt
      */
     protected NeoHubInfoResponse fromNeoHubFetchPollingResponse() {
         if (socket == null || config == null) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE);
```

Comment:
why are you removing this updateStatus?

Reply:
It is just a question of being consistent. The binding has two functions: a) polling/reading from the bridge, and b) issuing/writing commands to the bridge. The polling function reads states from the bridge, and updates the Thing and Channel states in OH accordingly. Whereas the command function responds to commands from OH and writes states to the bridge. As a general principle updateStatus should be done in the polling (read states) part of the code. The code that I modified here is part of the command (write states) function, so to be consistent it should NOT be changing the Thing status. It is not a big if we eliminate the updateStatus here, because the respective error will anyway always be picked up on the next polling cycle.